pkg/dbconn/tablelock.go

```diff
@@ -38,30 +45,24 @@ func NewTableLock(ctx context.Context, db *sql.DB, table *table.TableInfo, confi
 				}
 			}()
 			// We need to lock all the tables we intend to write to while we have the lock.
-			// So this is at least the table and the new table, but in the multi-subscription
-			// case it could be a lot more tables. If they are not locked, mysql will return
-			// an error when trying to write to them.
-			// TODO: accept a slice of tables to be locked as a parameter.
-			logger.Warnf("trying to acquire table lock, timeout: %d", config.LockWaitTimeout)
-			_, err = lockTxn.ExecContext(ctx, fmt.Sprintf("LOCK TABLES %s WRITE, `%s`.`_%s_new` WRITE",
-				table.QuotedName,
-				table.SchemaName, table.TableName,
-			))
```

Comment:
Wow didn't realize the _new table was hardcoded until now. Good cleanup to be explicit.